<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FoodMenuWindow</class>
 <widget class="QDialog" name="FoodMenuWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>898</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>510</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>859</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLineEdit" name="lineEditItemName_0_0"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditItemName_0_1"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditItemName_0_2"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditItemName_0_3"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditItemName_0_4"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditItemName_0_5"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>859</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLineEdit" name="lineEditItemPrice_0_0"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditItemPrice_0_1"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditItemPrice_0_2"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditItemPrice_0_3"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditItemPrice_0_4"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditItemPrice_0_5"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>859</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLineEdit" name="lineEditItemName_1_0"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditItemName_1_1"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditItemName_1_2"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditItemName_1_3"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditItemName_1_4"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditItemName_1_5"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>859</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLineEdit" name="lineEditItemPrice_1_0"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditItemPrice_1_1"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditItemPrice_1_2"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditItemPrice_1_3"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditItemPrice_1_4"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditItemPrice_1_5"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>859</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLineEdit" name="lineEditItemName_2_0"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditItemName_2_1"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditItemName_2_2"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditItemName_2_3"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditItemName_2_4"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditItemName_2_5"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>859</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QLineEdit" name="lineEditItemPrice_2_0"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditItemPrice_2_1"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditItemPrice_2_2"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditItemPrice_2_3"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditItemPrice_2_4"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditItemPrice_2_5"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>859</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <widget class="QLineEdit" name="lineEditItemName_3_0"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditItemName_3_1"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditItemName_3_2"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditItemName_3_3"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditItemName_3_4"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditItemName_3_5"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>859</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_8">
    <item>
     <widget class="QLineEdit" name="lineEditItemPrice_3_0"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditItemPrice_3_1"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditItemPrice_3_2"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditItemPrice_3_3"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditItemPrice_3_4"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditItemPrice_3_5"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FoodMenuWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FoodMenuWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
